eks_infra/env_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#/ Usage: ./env_setup.sh 
+
+set -e
+
+# aws caller identity
+aws sts get-caller-identity
+
+#aws-iam-authenticator version
+aws-iam-authenticator version
+
+#install kubectl
+curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.18.9/2020-11-02/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+mv ./kubectl $(which kubectl)
```

Comment:
`which kubectl` will only work if kubectl has already been installed beforehand. Why can this statement not move kubectl to `/usr/local/bin` or something directly?

Reply:
kubectl is already installed in codebuild image. Installing the new binary and moving it to `/usr/local/bin`, `kubectl` still pointed out to the older version, hence the above way of installation.

Reply:
Thanks! Since we'd also like human users of the EKS setup to be able to use the setup script, we'll need to account for the scenario where someone doesn't have `kubectl` installed beforehand, and is installing it for the first time by running this script. To cover this, I believe it would be better to have a backup statement of some kind to allow users to install kubectl without any inconvenience.

Reply:
Added checks for each command in env_setup if they exist. For kubectl, if it exist, we also check if the available version is equal to `v1.18` as it is the latest.